### DISCUSSION TITLE
Custom made flight based on GPS

### DISCUSSION THREAD

**Discussion by Martiosk** [albar965/littlenavmap#928] [Q&A]
Hello! Is it possible to add a gps-route into microsoft flight simulator, and render that specific flight into a "movie". A little background: I am a journalist making a story about an eagle  with a gps tracker. It would be awesome to visualize the flight path and use the elements in a digital feature story (without the cockpit view). But - I don't know if thats possible at all. Maybe someone here knows? Best, Martin

**Comment by albar965** [comment]
> Hello! Is it possible to add a gps-route into microsoft flight simulator, and render that specific flight into a "movie". A little background: I am a journalist making a story about an eagle with a gps tracker. It would be awesome to visualize the flight path and use the elements in a digital feature story (without the cockpit view). But - I don't know if thats possible at all. Maybe someone here knows? Best, Martin

LNM can record GPS routes into its logbook which you can export as GPX but I suppose this will not help you.
There are some third party tool for MSFS replay. Maybe one of them can use coordinate files.

You also need more than coordinates to replay a flight. Bank, yaw, pitch and probably more parameters are needed too.

**Reply by Martiosk** [reply]
I see - the only thing i got, is gps (with parameters like altitude speed etc). Do you know these third party tools? I have never even played MSFS, but i assume its the best flight sim out there (?). Best, Martin

**Reply by albar965** [reply]
There is this free one:
https://de.flightsim.to/file/8163/flight-recorder
and this commercial one:
https://flightcontrolreplay.wordpress.com/

I did not use these tools so far since I'm mostly flying X-Plane.
